pinecone/config/openapi.py

```diff
@@ -17,7 +17,7 @@
 class OpenApiConfigFactory:
     @classmethod
     def build(cls, api_key: str, host: str = None, **kwargs):
-        openapi_config = OpenApiConfiguration.get_default()
```

Comment:
What's the reason for this change?

Reply:
This method doesn't exist in the generated output of openapi 5.2. It shouldn't matter, since we're setting up all the important properties within this config factory class anyway.